### PR TITLE
Production Ready 

### PR DESCRIPTION
**Pull request by ArdusJax**
Initial production version 0.0.1

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/ArdusJax/ScalaClients/pull/4" target="_blank">ReviewNinja

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+language: scala
+scala:
+  - 2.11.7
```
Comment (ArdusJax) [inline .travis.yml:3]: Scala 2.11.7 is used.

File: app/com/ardusjax/webservice/clients/ArtifactoryWSC.scala [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+              subscriptions_url:String,
+              organizations_url:String,
+              repos_url:String,
+              events_url:String,
+              received_events_url:String,
+              `type`:String,
+              site_admin:Boolean
+             )
+
+  case class Repository(id:Int,
```
Comment (ArdusJax) [inline app/com/ardusjax/webservice/clients/ArtifactoryWSC.scala:102]: Here is the case class for Repository

File: build.sbt [hunk 3]
```diff
@@ -0,0 +1,17 @@
+name := "WebServiceClients"
+
+version := "1.0"
+
+lazy val `webserviceclients` = (project in file(".")).enablePlugins(PlayScala)
+
+scalaVersion := "2.11.7"
+
+libraryDependencies ++= Seq( jdbc , cache , ws   , specs2 % Test,
+  "org.webjars.npm" % "jquery" % "2.1.4",
```
Comment (ArdusJax) [inline build.sbt:10]: Why the npm webjar... that seems like a mistake, but good for now. Will need to look at it later.

File: public/javascripts/jquery-1.9.0.min.js [hunk 4]
```diff
@@ -0,0 +1,4 @@
...index:"tabIndex",readonly:"readOnly","for":"htmlFor","class":"className",maxlength:"maxLength",cellspacing:"cellSpacing",cellpadding:"cellPadding",rowspan:"rowSpan",colspan:"colSpan",usemap:"useMap",frameborder:"frameBorder",contenteditable:"contentEditable"},prop:function(e,n,r){var i,o,a,s=e.nodeType;if(e&&3!==s&&8!==s&&2!==s)return a=1!==s||!st.isXMLDoc(e),a&&(n=st.propFix[n]||n,o=st.propHooks[n]),r!==t?o&&"set"in o&&(i=o.set(e,r,n))!==t?i:e[n]=r:o&&"get"in o&&null!==(i=o.get(e,n))?i:e[n]},propHooks:{tabIndex:{get:function(e){var n=e.getAttributeNode("tabindex");return n&&n.specified?parseInt(n.value,10):At.test(e.nodeName)||jt.test(e.nodeName)&&e.href?0:t}}}}),kt={get:function(e,n){var r=st.prop(e,n),i="boolean"==typeof r&&e.getAttribute(n),o="boolean"==typeof r?Mt&&Ht?null!=i:Lt.test(n)?e[st.camelCase("default-"+n)]:!!i:e.getAttributeNode(n);return o&&o.value!==!1?n.toLowerCase():t},set:function(e,t,n){return t===!1?st.removeAttr(e,n):Mt&&Ht||!Lt.test(n)?e.setAttribute(!Ht&&st.propFix[n]||n,n):e[st.camelCase("default-"+n)]=e[n]=!0,n}},Mt&&Ht||(st.attrHooks.value={get:function(e,n){var r=e.getAttributeNode(n);return st.nodeName(e,"input")?e.defaultValue:r&&r.specified?r.value:t
```
Comment (ArdusJax) [inline public/javascripts/jquery-1.9.0.min.js:1]: Using old JQuery here, we'll need to pull in the new one for the next sprint.

**Comment by ArdusJax** [comment]
Looks like a good first production run. !complete !ninjastar

**Comment by ArdusJax** [comment]
!completed !ninjastar

**Pull request merged**